### PR TITLE
Fixes #904

### LINKED ISSUE #904: ZEC overwinter support
related https://github.com/LedgerHQ/ledger-wallet-chrome/pull/109

- [x] libcore integration
- [x] ledgerjs integration
- [x] live integration

### PR DESCRIPTION
**Pull request by meriadec**
fixes #904

### CODE REVIEW

File: src/commands/libcoreSignAndBroadcast.js [hunk 1]
```diff
@@ -119,7 +121,7 @@ async function signTransaction({
 
   const changePath = output ? output.getDerivationPath().toString() : undefined
   const outputScriptHex = Buffer.from(transaction.serializeOutputs()).toString('hex')
-  const lockTime = transaction.getLockTime()
+  const lockTime = undefined // TODO: transaction.getLockTime()
```
Comment (gre) [inline src/commands/libcoreSignAndBroadcast.js]: why is this commented out? could you explain a bit what it means? wouldn't this create a regression on some coin that need it? 🤔

Comment (meriadec) [reply src/commands/libcoreSignAndBroadcast.js]: seen with the libcore team: this field is not handled yet on their side, it always returns a default value, and we realized that it was what make Zcash tx fail :thinking: as it is not necessary for other coins, we choose to comment it for all, but if you think it's safer, we can set it `undefined` just for Zcash. nb: I tested on other coins (stratis, dash, ..)

Comment (gre) [reply src/commands/libcoreSignAndBroadcast.js]: so is ZCash working right now? 🤔

Comment (meriadec) [reply src/commands/libcoreSignAndBroadcast.js]: [yep](https://explorer.zcha.in/transactions/667e86a8b696a5cb60753d66ac1b900078e4e909f32da89e333afee3433984c3)

Comment (meriadec) [reply src/commands/libcoreSignAndBroadcast.js]: will add a comment in the PR. hodl on

**Pull request merged**